Rohit/PostgresAssignment/src/services/class.ts

```diff
@@ -0,0 +1,20 @@
+const classSql = require('../database/dbconnect')
+
+exports.get_classes = async () => {
+  try {
+    await classSql.query("set search_path to myschool");
+    return await classSql.query("select * from classes");
```

Comment:
use order by for all select queries. select returns random order rows.

Reply:
done